docs/log-collector/README.md

```diff
@@ -105,6 +108,42 @@ Verify installation with `kubectl tvk-log-collector --help`
 
         kubectl tvk-log-collector --clustered --keep-source-folder --log-level error
 
+- To collect logs by providing gvk :
+
+        kubectl tvk-log-collector --clustered --gvk '[{"group": "<group>", "version": "<version>", "kind": "<kind>"}, {...}]'
+
+- To collect logs by providing label-selector :
+        
+        kubectl tvk-log-collector --clustered  --label-selector '[{"matchLabels":{"key":"value","key2":"value2"}}]'
```

Comment:
same here.. Providing JSON string via flags is not user-friendly.
Refer implementation of `kubectl get -l` command.. use the same.

Reply:
done